src/content-scripts/canvas.js

```diff
@@ -38,21 +38,62 @@ var drawCanvasElement = function(canvasElement, data){
   };
 };
 
-var appendCanvasElement = function(user){
+var saveUserCanvas = function(){
+  var data = canvas.toDataURL();
+  ref.child(currentUser).set(data);
+  console.log('saving user canvas');
+};
+
+var drawLine = function(){
+  ctx.beginPath();
+  ctx.moveTo(prevX + pageXOffset, prevY + pageYOffset);
```

Comment:
Can someone verify that we can call pageXOffset/pageYOffset without using window.pageXOffset.